srv/salt/ceph/functests/1node/quiescent/init.sls

```diff
@@ -4,6 +4,13 @@
 {{ node }}:
   test.nop
 
+PGs greater than zero:
```

Comment:
Wondering what this change is aiming at. @swiftgist any insight?

Reply:
It's been a while since I did the original, but IIRC the tests are not valid for an empty cluster.  Running a test that says "active+clean" vs. "not active+clean" gives the admin/tester a false sense of success.

Reply:
So the solution is for the teuthology test to create an object before triggering the `ceph.functests.1node` orchestration?

Should the master version of this commit be reverted if we're not going to merge the backport?

Reply:
The normal exercise has been to run something in Stage 4, such as CephFS to create pools.  Then the cluster is no longer empty and the test is actually valid.